Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim-buster as base
+RUN apt-get update; apt-get install curl -y
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:$PATH"
+RUN mkdir todo_app
```

Comment:
`WORKDIR` will make folders that don't exist, so if you move that here you can skip this line